test/benchmark_http_client_test.cc

```diff
@@ -276,8 +297,38 @@ TEST_F(BenchmarkClientHttpTest, BadContentLength) {
   };
 
   EXPECT_CALL(stream_encoder_, encodeData(_, _)).Times(0);
-  testBasicFunctionality(1, 1, 1);
+  testBasicFunctionality(1, 1, 1, request_generator);
   EXPECT_EQ(1, getCounter("http_2xx"));
 }
-
+TEST_F(BenchmarkClientHttpTest, MultipleRequestsDifferentPath) {
+  std::vector<HeaderMapPtr> requests_vector;
+  std::initializer_list<std::pair<std::string, std::string>> header1{{":scheme", "http"},
+                                                                     {":method", "GET"},
+                                                                     {":path", "/a"},
+                                                                     {":host", "localhost"},
+                                                                     {"Content-Length", "1313"}};
+  std::initializer_list<std::pair<std::string, std::string>> header2{{":scheme", "http"},
```

Comment:
I don't think you need the initializer_list you can pass in the const array directly to the TestRequestHeaderMapImpl constructor.

Reply:
I don't think that works. I can make it a const initializer_list, which is done. But const array is not accepted. There is no default conversion between initializer_list and array that I am aware of.